src/worldcereal/openeo/feature_extractor.py

```diff
@@ -0,0 +1,115 @@
+"""Feature computer GFMAP compatible to compute Presto embeddings."""
+
+import xarray as xr
+from openeo_gfmap.features.feature_extractor import PatchFeatureExtractor
+
+class PrestoFeatureExtractor(PatchFeatureExtractor):
+    """Feature extractor to use Presto model to compute embeddings.
+    This will generate a datacube with 128 bands, each band representing a
+    feature from the Presto model.
+    """
+
+    import functools
+    from pathlib import Path
+    from typing import Tuple
+
+    PRESTO_PATH = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/presto.pt"  # NOQA
+    BASE_URL = "https://s3.waw3-1.cloudferro.com/swift/v1/project_dependencies"  # NOQA
```

Comment:
`project_dependencies`doesn't say much. And how is this now different from the ONNX dependencies?

Reply:
This is much more than ONNX as it contains pytorch and all the other presto dependencies